### ISSUE TITLE
Arm64 alpine images

### ISSUE THREAD

**Issue by andytson-inviqa** [jenkinsci/docker#1244]
### Feature Request

Support arm64 alpine images

Was there anything blocking this in the work for multi-arch support?

**Comment by timja** [comment]
Yes support upstream, arm64 alpine port was only merged into Java 16 which the Jenkins project doesn't officially support.

It's been backported into Zulu and Bellsoft Java 11 which are different vendors to what we normally use but possibly could be done if someone were to PR it.
Eclipse Temurin which is our standard distribution has said they will backport it to 11 but I don't think it has happened yet

**Comment by MarkEWaite** [comment]
As of May 1, 2022, Eclipse Temurin only provides amd64 images for [Alpine images](https://hub.docker.com/_/eclipse-temurin?tab=tags&page=1&name=11.0.15_10-jdk-alpine).  Still blocked by lack of an upstream image to support it.

**Comment by timja** [comment]
this is possible now, not blocked on upstream anymore

**Comment by marvinruder** [comment]
As announced [here](https://github.com/adoptium/containers/issues/158#issuecomment-1758310633) and seen [here](https://hub.docker.com/_/eclipse-temurin/tags?page=1&name=alpine), Eclipse Temurin’s JDK21 Alpine images are now available for arm64 architecture.

**Comment by lemeurherve** [comment]
Reopening this issue until the next weekly release with the official Eclipse Temurin images as base, providing arm64 architecture.

**Comment by lemeurherve** [comment]
Forgot to close this issue, alpine arm64 images are available.
